plover/suggestions.py

```diff
@@ -0,0 +1,52 @@
+import collections
+
+Suggestion = collections.namedtuple('Suggestion', 'text steno_list')
+
+
+class Suggestions(object):
+    def __init__(self, dictionary):
+        self.dictionary = dictionary
+
+    def list(self, translation):
```

Comment:
Is this shadowing? It will only be accessible by self.list and on instances. Curious if there's a case where you'd actually get a conflict with the built in.